src/infrastructure/TransactionCreationParams.ts

```diff
@@ -7,7 +7,7 @@ interface TransactionCreationParams {
 
 interface TransferTransactionCreationParams extends TransactionCreationParams {
     recipientAddress: string;
-    mosaicDetails: Array<{namespaceId: string, amount: number}>;
+    mosaicDetails: Array<{mosaicId?:string, namespaceId?: string, amount: number}>;
```

Comment:
wouldnt it make more sense to use the namespace name? 

this `new NamespaceId('dhealth.dhp')` gives you the namespace ID in return

Reply:
> wouldnt it make more sense to use the namespace name?
> 
> this `new NamespaceId('dhealth.dhp')` gives you the namespace ID in return

Actually this is the namespace name, sorry I named the param ambiguously, will update now

Reply:
In the commit fe30606648133b99d510b0414dc42d2098451454, namespaceId was renamed to namespaceName to avoid confusion, PTAL